eclipse-collections-testutils/src/main/java/org/eclipse/collections/impl/test/Verify.java

```diff
@@ -65,28 +64,28 @@
 import org.eclipse.collections.impl.tuple.ImmutableEntry;
 import org.eclipse.collections.impl.utility.ArrayIterate;
 import org.eclipse.collections.impl.utility.Iterate;
-import org.junit.Assert;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNotSame;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertSame;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import org.junit.jupiter.api.Assertions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotSame;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
 
 /**
- * An extension of the {@link Assert} class, which adds useful additional "assert" methods.
+ * Adds useful additional "assert" methods.
  * You can import this class instead of Assert, and use it thus, e.g.:
  * <pre>
  *     Verify.assertEquals("fred", name);  // from original Assert class
  *     Verify.assertContains("fred", nameList);  // from new extensions
  *     Verify.assertBefore("fred", "jim", orderedNamesList);  // from new extensions
  * </pre>
  */
-public final class Verify extends Assert
+public final class Verify extends Assertions
```

Comment:
I'd like to get input from @donraab. This is a source breaking change, only for users of Verify, in order to move from JUnit 4 to JUnit 5. I think it makes sense to do. I'd like to think through together if we've done everything to make the source breaking changes as small as possible.

Reply:
@motlin I'm ok with this in theory. Hard to tell what issues this might run into in practice. What might be better, although would cause more work and potentially source breaks would be to break the inheritance relationship altogether. This means Verify can't be used as a means to automatically route calls to Assert or Assertions, but also means there won't be a similar issue when JUnit 6 happens.

Reply:
I dread the amount of work it would be, but I'd like to "override" every single method, delegate from Verify to Assert or Assertions, and mark the new overrides as deprecated and inlineable.